docs/release-notes/release-notes-0.13.4.md

```diff
@@ -16,15 +16,13 @@ with 'make go-build-noui' and 'make go-install-noui'](https://github.com/lightni
 
 ### LND
 
```

Comment:
LND also updated yeah?

Reply:
Hmm yeah, I believe that's already in the release notes :)! Let me know if I'm misunderstanding your feedback though :)

Reply:
my bad! was not viewing the side by side diff 🙈